core/src/main/scala/org/apache/spark/scheduler/cluster/mesos/MesosSchedulerBackendUtil.scala

```diff
@@ -119,12 +122,14 @@ private[mesos] object MesosSchedulerBackendUtil extends Logging {
   }
 
   /**
-   * Setup a docker containerizer
+   * Setup a docker containerizer from SparkConf settings
    */
   def setupContainerBuilderDockerInfo(
     imageName: String,
     conf: SparkConf,
     builder: ContainerInfo.Builder): Unit = {
+    val forcePullImage = conf
+      .getBoolean("spark.mesos.executor.docker.forcePullImage", defaultValue = false)
```

Comment:
We don't really spell out the parameter name in other places in this call. I recommend just putting the value here.